install.yaml

```diff
@@ -26,7 +27,6 @@ post_install_actions:
   - cp core-dev/gitignore ../.gitignore
   - mkdir -p ../test_output
   - chmod +w ../test_output
-  - ddev exec corepack enable
```

Comment:
I would readd `- ddev exec corepack enable`that way it is ensured that folks not upgraded to 1.23.0 are able to to run the addon as well. folks on 1.23.0 have things already set in their projects config.yaml